Sources/momento/messages/responses/topics/TopicSubscribe.swift

```diff
@@ -0,0 +1,33 @@
+import GRPC
+import Foundation
+
+public protocol TopicSubscribeResponse {}
+
+@available(macOS 10.15, iOS 13, *)
+public class TopicSubscribeSuccess: TopicSubscribeResponse {
+    public let subscription: AsyncCompactMapSequence<GRPCAsyncResponseStream<CacheClient_Pubsub__SubscriptionItem>, TopicSubscriptionItemResponse>
+    
+    init(subscription: GRPCAsyncResponseStream<CacheClient_Pubsub__SubscriptionItem>) {
+        self.subscription = subscription.compactMap(processResult)
+    }
+}
```

Comment:
The `GRPCAsyncResponseStream` returned from the `subscribe` call is already an [`AsyncSequence`](https://developer.apple.com/documentation/swift/asyncsequence). The [`compactMap`](https://developer.apple.com/documentation/swift/asyncsequence/compactmap(_:)-gfdq) call allows us to transform the grpc responses (heartbeats, items, etc) into Momento `TopicSubscriptionItemResponse` objects and omits the `nil` items in the transformed sequence.